### PR TITLE
fix pin create; add tags relation

### PR DESCRIPTION
**Pull request by sokolikp**

### CODE REVIEW

File: Sweeper/Models/Pin.swift [hunk 1]
```diff
@@ -12,16 +12,50 @@ import Parse
...
+        }
+    }
     
     static func parseClassName() -> String {
         return "Pin"
     }
+    
+    func setLocation() {
+        if latitude != nil && longitude != nil {
+            location = PFGeoPoint(latitude: latitude as! Double, longitude: longitude as! Double)
```
Comment (huw30) [inline Sweeper/Models/Pin.swift]: Nice Job!

File: Sweeper/Models/Tag.swift [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+//  Copyright © 2017 team11. All rights reserved.
+//
+
+import Foundation
+import Parse
+
+class Tag: PFObject, PFSubclassing {
+    @NSManaged var name: String?
+    
+    static func parseClassName() -> String {
```
Comment (huw30) [inline Sweeper/Models/Tag.swift:15]: Nice!

File: Sweeper/Services/AWSS3Service.swift [hunk 3]
```diff
@@ -8,17 +8,17 @@
 
 import AWSS3
 
-class AWSS3Client {
+class AWSS3Service {
```
Comment (huw30) [inline Sweeper/Services/AWSS3Service.swift:11]: Thanks :)

File: Sweeper/Services/PinService.swift [hunk 4]
```diff
@@ -11,41 +11,57 @@ import UIKit
...
-}
+var _pins: [Pin] = [Pin]()
 
 class PinService {
     static let sharedInstance = PinService()
     
-    func create(pin: Pin, completion: @escaping (Bool, Error?) -> Void) {
-        pin.saveInBackground { (success: Bool, error: Error?) in
-            completion(success, error)
+    func create(pin: Pin, withImage: UIImage?, tagNames: [String]?, completion: @escaping (Bool, Error?) -> Void) {
```
Comment (huw30) [inline Sweeper/Services/PinService.swift:19]: Definitely should belong here 👍

**Review by huw30 — APPROVED** [review]
LGTM!

**Comment by wumingx89** [comment]
LGTM 🚢

**Pull request merged**